add: New Station "Nova"

PiroMage opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений
<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

PiroMage committed:
New Station "Nova"

Den109G commented on code/game/objects/effects/decals/syndie_logo_decals.dm:
```diff
@@ -11,3 +11,24 @@
+	icon = 'icons/obj/NovaStation.dmi'
+	icon_state = "NStation1"
+	layer = TURF_LAYER
+	desc = "Glory to Nanotrasen!"
```
нужна пустая строка в конце файла.
```suggestion
	desc = "Glory to Nanotrasen!"

```

Den109G commented on code/modules/mob/living/simple_animal/hostile/retaliate/carp.dm:
```diff
@@ -0,0 +1,22 @@
+	melee_damage_upper = 20
+	unique_pet = TRUE
+	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 2, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+	gender = MALE
```
нужна пустая строка в конце файла.
```suggestion
	gender = MALE

```

PiroMage committed:
Пустые строчки

Den109G commented:
Если ПР не готов (станции как я вижу нет) - можешь перевести этот ПР в драфт для доработок. Где-то справа есть кнопка для этого.

PiroMage committed:
Сама карта и лаваленд

PiroMage committed:
Update syndie_logo_decals.dm

PiroMage committed:
Первые правки света, лестниц и другое.

PiroMage committed:
Вторые правки света

PiroMage committed:
Следующая волна правок карты

PiroMage committed:
Переработка карго, мусорки, меда, отбытия.

Переработка мусоропровода карго и перенос с изменением планировки мусорки.
Изменение отбытия.
Перенос с перепланировкой комнаты отдыха медотдела.

PiroMage committed:
Изменение коморки уборщика, техов, атмоссии.

Перестройка коморки уборщика, прилегающих к нему техов.
Добавление системы обработки и выкачки газа из камеры смешивания в атмосферном отделе.

PiroMage committed:
Небольшие правки света, проводки, медотдела, отбытия.

PiroMage committed:
Изменение описания статуй у меда

PiroMage committed:
Перестройка медотдела, дополнение техов.

Перестроена комната отдыха, первое и второе хранилище медотдела, химия меда, вирусология, комната с криокапсулами. Дополнение северо-западных техов второго этажа.

PiroMage committed:
Обновление вирусологии, добавление постера

PiroMage committed:
Исправление проводки трубы в техах, обновление кода постера

PiroMage committed:
Небольшие правки дыр космоса и отделов

Daeberdir committed:
add: Jetpack for mices. (#5176)

* add: mouse idle animation

* fix: daebdir review

* 1/2

* Done.

* Conflicts.

* Cleaning.

* Oh shit here we go again.

* A-a--a-a-a-a-aa-a--a-a-a-and done.

* Update mobs.dm

* Eh...

* Better that.

* Zwei corrections.

* Unnecessary message.

* Lesenka R.I.P 2024 - 2024

* Conflict resolving.

* Conflict resolve.

* `update_icons()` using. Random rodent spawners.

* And done.

---------

Co-authored-by: ROdenFL <144662735+ROdenFL@users.noreply.github.com>

ItsMeJaba committed:
add: Central Command rework (#3900)

* FerryPack

* в догонку

* Промежуток №2

* Проба дроппода

* Ребейз карты ручками

* Промежуток 3

* Промежуток 4

И подчистка

* 3

* Update CentComm.dmm

* 4

* Фиксы

* Мерж ручками

Перенес ручками все, что было добавлено во время простоя + заметил, что правую часть корабля скосило

* АААА

АААА

* AAAA

* ААА

Я идиот.

* Э, а тут что-то происходит?

* Мерж конфликт кривой жабьей лапкой

* Ой

Я что-то сделал неделю назад и уже не помню что

* Mapper's ref and something what i cant remember

* Final?

Я уже не знаю чего мне добавлять, поправил что мог, навел марафета, раскидал мин под кровати

* Conflict.net

Решаем ручками конфликты

* Bugfix.da

* Bugfix.da

Ahaha, 3pm fixes

Bugfix.da

Bugfix.da

Bugfix.da

* Remove BSA fun

Снес стекло, которое потенциально могло бы помочь боргам стрелять из БСА, находясь в ресторанобарокафе

* Мержконфликты

Слава ганпорно, больше подтипов 1 пушки ради скина, БОЛЬШЕ

* Little nit of hehe

Правки после ж0ского ревью бригадира

* LastFIxes

Заменил открытые говнокодом VV шаттерсы на аналог с говнокода
Нашел потерянные пару тайлов под движками ферри (всех 16)

* Ваши икон заборонено

Апдейт иконок не дал мне своровать спрайты для моих костыльно-кастомных вендоматов

* Sho to

* Актуализация

Я просто вырезал кусок с НТ ЦК, внес изменения за эти... 5 месяцев? И воткнул на актуальную карту

* Актуализация

Я просто вырезал кусок с НТ ЦК, внес изменения за эти... 5 месяцев? И воткнул на актуальную карту

* Update CentComm.dmm

* Fix: overlays & smooth criminal

Поправил оверлеи добавленных вендоматов, поправил шаттерсы-стены-костыли над туррелями

* logo fix

Из-за наслоения прерывался спрайт надписи

* fix: bump tele

После рефактора звея с выпилом прока бампа он начал есть фальш-стены, Д2 больше не покинет оружейной Кеннеди

* "Фикция"

Доебы мисстера SQUEEK учтены

* Refucktor

Спасибо за рефактор конвейеров, было невкусно и грустно

* Мои любимые рефаторы с изменениями пути

* Мерге конфликт

Как же я люблю апдейты на +-/слово/ в пути

PiroMage committed:
Изменение наименования центрального коридора на первом этаже

SAADf603 commented:
[Ревью](https://discord.com/channels/617003227182792704/943940908162875473/1267705571952300146)
![image](https://github.com/user-attachments/assets/28b34e31-26a5-40fe-b2d6-675320191a49)

SAADf603 commented:
Ловушка рефактора:
![image](https://github.com/user-attachments/assets/26e9be33-89ac-45ba-9f81-38db3ec7eb8f)
```мышь/серая``` теперь просто ```мышь```
```шкаф/барный``` теперь ```шкаф/защищённый_шкаф/барный``` (то же самое должно быть с детективом)

помимо ловушки рефактора:
![image](https://github.com/user-attachments/assets/81975250-0093-4cfc-ad30-612a5c5b2422)
насрано латисами где попало и неверно указан путь зоны ```строительная/коридор```

PiroMage committed:
Минус часть атмоса в стенах. Починка бластдоров ксено и т.д.

PiroMage committed:
Что-то стало лучше №1

1) Исправление части атмоса.
2) Убрана проводка и трубы с центральных переходов в стороны от стеклянного пола ВО ИМЯ КРЫСОТЫ.
3) Переименованы этажи для удобства просмотра камер.
4) Что-то еще...

PiroMage committed:
Что-то стало лучше №1

1) Исправление части атмоса.
2) Убрана проводка и трубы с центральных переходов в стороны от стеклянного пола ВО ИМЯ КРЫСОТЫ.
3) Переименованы этажи для удобства просмотра камер.
4) Что-то еще...

PiroMage committed:
Исправленные вручную трубы мусоропровода

SAADf603 commented on code/modules/map_fluff/nova.dm:
```diff
@@ -0,0 +1,18 @@
+	company_name  = "Nanotrasen"
+	company_short = "NT"
+	starsys_name  = "Epsilon Eridani"
+	webmap_url = "https://webmap.affectedarc07.co.uk/maps/ss1984/celestation/"
```
```suggestion
	webmap_url = "https://webmap.affectedarc07.co.uk/maps/ss1984/nova/"
```
но это ещё нужно будет сделать после открытых тестов

SAADf603 commented on code/modules/map_fluff/nova.dm:
```diff
@@ -0,0 +1,18 @@
+	traits = list(
+	list(MAIN_STATION, STATION_LEVEL = "First Floor", STATION_CONTACT, REACHABLE, AI_OK, ZTRAIT_UP),
+	list(STATION_LEVEL = "Second Floor", STATION_CONTACT, REACHABLE, AI_OK, ZTRAIT_DOWN, ZTRAIT_BASETURF = /turf/simulated/openspace),
+	)
```
```suggestion
	)
	admin_only = TRUE //для первых тестов
```

PiroMage committed:
Нерф лута черного рынка, починка лаваленда

PiroMage committed:
Фикс пола черного рынка. Правки труб.

PiroMage committed:
Фикс пола черного рынка. Правки труб.

PiroMage committed:
Правки атмоса.

PiroMage committed:
Правки атмоса.

PiroMage committed:
Исправление рандом-спавнеров и атмоса.

PiroMage committed:
Что-то стало лучше №2

1) Починка терминалов под полом АПЦ.
2) Замена рабочих АПЦ в покинутых отделах на нерабочие.
3) Убраны изольки в скрытый шкаф в старом морге.
4) Вернул культистский меч, болу, точ.камень в сейф, но перенес в старую церковь под плитку.
5) Убрал космос с лестницы в южных техах.

PiroMage committed:
Что-то стало лучше №3

1) Добавил лампочек в покинутый бойцовский клуб.
2) Поменял аптечку в медпункте аванпоста лаваленда с пустой на заполненную...

PiroMage committed:
Лаваленд стал лучше

1) Починил трубопровод на аванпосте шахтеров.
2) Закрыл дыру под бухломатом, ведущую в космос.
3) Починил терминалы под полом АПЦ каторги.

PiroMage committed:
Что-то стало лучше №4

1) Замена некорректно работающих тинтовых окон в приватной комнате у НТРа на ставни.
2) Замена некорректно работающих тинтовых окон в приватной комнате у Психолога на ставни.
3) Добавление скрытого сейфа с бухлом под плитку пола за барной стойкой в старом казино.

PiroMage committed:
Что-то стало лучше №5

Вернул паралайз окна в приватные комнаты психолога и НТРа, оставив шаттерсы на отдельной кнопке.

PiroMage committed:
Исправление радиуса интеркомов и камер.

Плюсом дополнил c_tag'ами камеры, без них были нерабочими.

PiroMage committed:
Люблю интеркомы

Закрыл последние дырки с зонами, не охваченными интеркомами.

PiroMage committed:
Возвращение пета ГСБ и постеров.

PiroMage committed:
Revert "add: Jetpack for mices. (#5176)"

This reverts commit c86554a2fd9140bb44aeabf24c8b0d3362a09103.

PiroMage committed:
Revert "hotfix: centcom rework (#5583)"

This reverts commit 0d43e1d17bbbf7a30b09c9e1e526d01456b66637.

PiroMage committed:
Reapply "hotfix: centcom rework (#5583)"

This reverts commit 14c426e522fdfb6fd9f9cc8a5917a8e15a36269a.

PiroMage committed:
Правки кода

PiroMage committed:
Revert "add: Central Command rework (#3900)"

This reverts commit 810b4adcebb9fa9cbe9774cad5938d1eded65261.

PiroMage committed:
Reapply "add: Central Command rework (#3900)"

This reverts commit 84dd99671a589dd7b7445b7117df156591a205ed.

PiroMage committed:
Revert "add: Central Command rework (#3900)"

This reverts commit 810b4adcebb9fa9cbe9774cad5938d1eded65261.

PiroMage committed:
Reapply "add: Central Command rework (#3900)"

This reverts commit 98c63cf2a96710939aed78c34bdd09240c11f417.

Daeberdir committed:
Fix teaching (I'm not the best teacher).

PiroMage committed:
Правки кода

PiroMage committed:
Обновление кода карты

PiroMage committed:
НаноКарта

Daeberdir committed:
Nanomap initial. (#1)

PiroMage committed:
Что-то стало лучше №6

1) Убраны провода под вентами/скрабберами
2) Убраны тупиковые и не подключенные трубы.
3) Добавлены трубы на второй этаж механика.
4) Исправление кода постеров и перестройка их очередности.
5) Убрано лишнее лого НТ.

PiroMage committed:
Что-то стало лучше №7

1) Правка пета ГСБ - карпа, теперь он общается через "gnashes"
2) Правки света, перемещение интеркомов и другой машинерии к более удобным позициям.

PiroMage committed:
Что-то стало лучше №8

1) Изменение кода статуи ангела у священника.
2) Убраны провода под вентами/скрабберами.
3) Уменьшение света в техах и некоторых комнатах отделов.
4) Добавлены маленькие вентиляторы под аирлоки шаттлов, дабы не нагружать атмос.

PiroMage committed:
Что-то стало лучше №10

1) Изменение прибытия и точек прилета ПОДов.
2) Изменение крепления камер, дабы не наслаивались на лампы (Где возможно было поменять).
3) Изменение наномапы первого и второго этажа.
4 Что-то еще.

PiroMage committed:
Восстановление карты

Восстановление карты

PiroMage committed:
Восстановление карты

Восстановление карты

PiroMage committed:
Внесение обновленного робо и подпилотской

1) Добавлены робоквесты на карту.
2) Добавлена винтовка ПОД-пилоту в кабинет.

PiroMage committed:
Добавление карпа

PiroMage committed:
Повторное добавление карпа

PiroMage committed:
Восстановление прежней версии карты

PiroMage committed:
Починка кода

PiroMage committed:
Починка шаттлов

PiroMage committed:
Починка постеров и шаттла

PiroMage committed:
Починка карт

PiroMage committed:
Починка животных

PiroMage committed:
Починка карты

Убран эффект инициализации, вероятно, с маркеров шахтерских

PiroMage committed:
Небольшая перестановка фаералярма

PiroMage committed:
Замена пола в баре

PiroMage committed:
Улучшение карты

1) Добавлен переход между вторым этажом механика и техами мостика.
2) Дополнительные выходы в космос в переходе механика.
3) Добавлены решетки, где они должны быть, убраны там, где они выглядят неподходяще.
4) Добавлена комнатка в техи медотдела.
5) Изменена дверь в театре с ботанической на дверь с общим доступом.

PiroMage committed:
Изменение карты и нанокарт

Bizzonium commented:
Сделаешь ребейз или мерж новых коммитов? Если тебе будет проще сделать апдейт через мерж, то можно и так, теперь это не так критично.

Daeberdir commented:
Всё же стоит дотерпеть до момента, когда я смогу сказать, что мапа готова к админским тестам. Если нужно это приоритезировать - сделаю.

Bizzonium commented:
Просто думал на тестмерж пустить

Daeberdir commented:
Ошибки Фаррагуса мы повторять не будем. Первое впечатление игроков на релизе именно **станционной** мапы - очень важно.

PiroMage committed:
Добавление зон и улучшение карты

1) Добавлены свои зоны для туалета прибытия, театра.
2) Заместо названия Break Room в медотделе теперь зона Med Restroom.
3) Заместо названия Hallway на саттелите теперь зона Ai satellite exterior hallway
4) Заменены большие фаерлоки у лестниц на их мелкий и более презентабельный аналог.

Daeberdir committed:
Brigadier's tweaks.

- Куча всего. Что не было лень - ушло к Пиро в ЛС.
- Нужна ещё пачка доработок.

PiroMage committed:
Улучшение карты

PiroMage committed:
Доп улучшение карты

PiroMage committed:
Улучшение карты

PiroMage committed:
Перестройка мехбея робо, токсинной.

Попозже изменю хирургичку робо для КПБ и КРБ

PiroMage committed:
Изменение операционной робо и старого морга.

Bizzonium committed:
Merge branch 'master220' into NovaStation

PiroMage committed:
Доп.пробел к листам кода карты

Bizzonium commented on code/modules/map_fluff/nova.dm:
```diff
@@ -0,0 +1,19 @@
+// Made by PiroMage (https://github.com/PiroMage)
+
+/datum/map/Nova
```
```suggestion
/datum/map/nova
```

Bizzonium committed:
server: add CI

PiroMage committed:
Merge branch 'NovaStation' of https://github.com/PiroMage/Paradise into NovaStation

PiroMage committed:
Правки труб
